pkg/build/builder/cmd/builder.go

```diff
@@ -318,7 +317,6 @@ func runBuild(out io.Writer, builder builder) error {
 
 // RunDockerBuild creates a docker builder and runs its build
 func RunDockerBuild(out io.Writer) error {
-	realglog.V(5)
```

Comment:
@bparees This got rid of the `realglog` import - did you intend to keep this?

Reply:
hm.  i have no idea what that was doing, so i guess not.....